apollo-federation/src/sources/connect/models.rs

```diff
@@ -198,23 +205,29 @@ impl Connector {
                     .get(schema)
                     .map(|f| f.ty.inner_named_type())
                 else {
-                    return Ok(None);
+                    return Err(internal_error!(
```

Comment:
Super pedantic, but maybe a `Result::map_err` could be used here and then no need to do the whole `let Ok(output_type) = ...`.